ethereum-consensus/src/deneb/presets/mod.rs

```diff
@@ -7,4 +7,9 @@ pub struct Preset {
     pub max_blobs_per_block: usize,
 }
 
-pub const TRUSTED_SETUP_JSON: &str = include_str!("trusted_setup.json");
```

Comment:
it looks like we no longer need to load from JSON here as the relevant data is provided from `c-kzg` now

can you remove this `trusted_setup.json` file as well?